contrib/libzookeeper/src/mt_adaptor.c

```diff
@@ -441,6 +441,8 @@ void *do_io(void *v)
         // if it is_unrecoverable()
         if(is_unrecoverable(zh))
             break;
+		
+		rc += 0; // silent "variable not used" warning
```

Comment:
Now it is "dead store" and "expression without effect". More smart compiler and static analyzer could complain.
To fix it, just remove variable.